python/ray/autoscaler/commands.py

```diff
@@ -199,9 +199,17 @@ def get_or_create_head_node(config, config_file, no_restart, restart_only, yes,
                 config["cluster_name"])
             provider.create_node(config["head_node"], head_node_tags, 1)
 
+        start = time.time()
```

Comment:
You want to use a monotonic clock to measure the passage of time, not a wall clock.
You can do this via `timeit.default_timer` in Python.